### DISCUSSION TITLE
early project too old - how to download earlier version of PK-Sim

### DISCUSSION THREAD

**Discussion by HLyster** [Open-Systems-Pharmacology/Forum#1491] [How-Tos]
Hi I am trying to open a project which was built in an earlier version of PK-SIm 6.1, should I download an earlier version - how do I do this and keep current version (am working on other projects)?

Many thanks
Haifa
![image](https://user-images.githubusercontent.com/36486243/228883418-836191bb-0517-4a8e-afa4-952a769541c7.png)

**Comment by StephanSchaller** [comment]
@Yuri05 ?

MAybe try the portable version of the first version released here on GitHub (installing the portable version is basically a download and you can execute the software locally)
scroll dow all the way here:
https://github.com/Open-Systems-Pharmacology/PK-Sim/releases

**Reply by HLyster** [reply]
thank you will try that
